keymaps/terminus.cson

```diff
@@ -21,6 +21,7 @@
 '.platform-darwin .terminus .terminal':
   'cmd-c': 'terminus:copy'
   'cmd-v': 'terminus:paste'
+  'cmd-delete': 'terminus:delete-line'
 
 '.platform-linux .terminus .terminal, .platform-win32 .terminus .terminal':
```

Comment:
You should add it to linux and win32 as `'ctrl-delete': 'terminus:delete-line'`:

Reply:
I thought about that, but i believe generally ctrl-delete reserved for deleting word on windows and linux machines. Wouldn't there be a clash of shortcuts?

Reply:
If we are going for familiar key bindings shouldn't mac keystrokes be <kbd>Ctrl</kbd> + <kbd>u</kbd> to delete current command?

Reply:
> Wouldn't there be a clash of shortcuts?

This is what I wanted to check and add, but if you have time to investigate and add, please =)